### DISCUSSION TITLE
Storing pictures in the Database

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#8725] [Help]
HI Guys:

The database I am working on requires that I be able to store pictures (in this case of used cars to prove that my drivers didnt damage them) Im using mysql -- I have the fields created on the table and set to type BLOB -- I am trying to figure out how to create the upload spot on the new row screen so that my drivers can upload the pics -- not really sure what I should be doing -- the image component does not seem to be doing it -- can someone help me or point me to a resource 

Thanks
TIM

**Comment by melohagan** [comment]
Hey Tim,

Unfortunately the external datasources do not support attachments: https://github.com/Budibase/budibase/issues/4302

You may need to handle the uploaded images through an internal Budibase DB.
